devel/geoipupdate/Portfile

```
@@ -1,37 +1,138 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
-PortGroup       github 1.0
-PortGroup       legacysupport 1.0
+PortSystem          1.0
+PortGroup           golang 1.0
 
-github.setup    maxmind geoipupdate 3.1.1 v
-revision        1
-categories      devel net
-maintainers     nomaintainer
-description     Maxmind geoip databse updater
-license         GPL-2+
+go.setup            github.com/maxmind/geoipupdate 6.0.0 v
+revision            0
+categories          devel net
+platforms           darwin
+maintainers         nomaintainer
+description         Maxmind geoip databse updater
+license             {apache mit}
```

Looks like this project is Apache 2.0 and MIT:

```suggestion
license             {Apache-2 MIT}
```

You can find the list of valid license values here: https://trac.macports.org/wiki/PortfileRecipes#licensekeyword

---

Updated